### ISSUE TITLE
Feature Request: Render mermaid html output

### ISSUE THREAD

**Issue by louagej** [shd101wyy/vscode-markdown-preview-enhanced#131]
When generating an html file from a .md file that includes a mermaid diagram, the diagram is displayed as text:
![2018_08_10_ 000](https://user-images.githubusercontent.com/7229592/43950445-9831b0a0-9c90-11e8-8ef9-862bc993a23a.png)

If you could add the two script lines in the html & rename the language-mermaid class to mermaid, the problem should be solved:
`<!DOCTYPE html> `
`<html> `
`<head> `
`<script src="https://cdnjs.cloudflare.com/ajax/libs/mermaid/7.1.2/mermaid.min.js"> `
`</script<script>mermaid.initialize({startOnLoad:true});</script> `
`... `
`<body> `
`<pre> `
`<code class="mermaid"> `
![2018_08_10_ 001](https://user-images.githubusercontent.com/7229592/43950488-b01df5b6-9c90-11e8-89ec-5225ab86133e.png)

**Comment by shd101wyy** [comment]
No, it should require you to insert `<script>`, because when you output html, the rendered `svg` image from preview is supposed to be included in html.
It might be a bug. Let me investigate. Thank you.

**Comment by louagej** [comment]
Hi,

The problem was caused by another mardown extension.
Altough the extension was diabled, I could still select the command from the command prompt in Visual Studio Code (recently used commands).
![2018_08_12_ 001](https://user-images.githubusercontent.com/7229592/43999607-5316af4e-9e10-11e8-8aaa-041d41d5a403.png)
So I removed my 'recently used commands' in VSC and this command doesn't show in the command prompt...
After that I'm only capable of right-clicking on the preview and choose one of the html methods.
Now the images are rendered correctly.
![2018_08_12_ 000](https://user-images.githubusercontent.com/7229592/43999611-6934671c-9e10-11e8-83bb-36ec432542df.png)

Regards, Job
